cms/middleware/toolbar.py

```diff
@@ -40,7 +40,7 @@ def is_cms_request(self, request):
 
         try:
             match = resolve(request.path_info)
-        except:
+        except Resolver404:
```

Comment:
This is fine but I think this needs to be listed somewhere as a possible breaking change as this handles every request and may have been swallowing other errors.

Reply:
Yeah that's a fair point. One for the release notes & us to remember perhaps?